com.unity.render-pipelines.universal/Documentation~/InstallingAndConfiguringURP.md

```diff
@@ -5,6 +5,6 @@ To use the Universal Render Pipeline (URP), you can start a new Project or upgra
 - [Create a new URP Project from a Template](creating-a-new-project-with-urp.md). If you are starting a new Project from scratch, this is the best choice. When you do this, Unity automatically installs and configures URP for you.
 - [Install URP into an existing Unity Project](InstallURPIntoAProject.md). If you have started a Project using the Built-in Render Pipeline, you can install URP and configure your Project to use URP. When you do this, you must configure URP yourself. You will need to manually convert or recreate parts of your Project (such as lit shaders or post-processing effects) to be compatible with URP.
 
-**Note:** URP's integrated post-processing solution does not currently support custom post-processing effects. If your Project uses custom post-processing effects, these cannot currently be recreated in URP's integrated post-processing solution. Custom post-processing effects will be supported in a forthcoming release of URP.
+**Note:** URP does not currently support custom post-processing effects. If your Project uses custom post-processing effects, these cannot currently be recreated in URP. Custom post-processing effects will be supported in a forthcoming release of URP.
```

Comment:
It's supported if used in PPv2. It's only the new built-in stack that doesn't support it yet.

Reply:
Thanks, I've edited this paragraph in the new commit.